source/dotnet/Library/AdaptiveCards/AdaptiveTextBlock.cs

```diff
@@ -108,5 +108,15 @@ public AdaptiveTextBlock(string text)
 #endif
         [DefaultValue(0)]
         public int MaxLines { get; set; }
+
+        /// <summary>
+        ///   The maximum width of the textblock
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+#if !NETSTANDARD1_3
+        [XmlAttribute]
+#endif
+        [DefaultValue(0)]
+        public uint MaxWidth { get; set; }
```

Comment:
Although `uint` is better, we used `int` previously as seen in `MaxLines` - maybe better to stay consistent